core/shared/src/main/scala/fs2/Pull.scala

```diff
@@ -116,20 +113,19 @@ object Pull extends PullLowPriority {
   def loop[F[_], O, R](using: R => Pull[F, O, Option[R]]): R => Pull[F, O, Option[R]] =
     r => using(r).flatMap { _.map(loop(using)).getOrElse(Pull.pure(None)) }
 
-  private def mapOutput[F[_], O, O2, R](p: Pull[F, O, R])(f: O => O2): Pull[F, O2, R] =
-    Pull.fromFreeC(p.get[F, O, R].mapOutput(f))
-
   /** Outputs a single value. */
   def output1[F[x] >: Pure[x], O](o: O): Pull[F, O, Unit] =
     new Pull(Algebra.output1[O](o))
 
   /** Outputs a chunk of values. */
   def output[F[x] >: Pure[x], O](os: Chunk[O]): Pull[F, O, Unit] =
-    if (os.isEmpty) Pull.done else fromFreeC(Algebra.Output[O](os))
+    if (os.isEmpty) Pull.done else new Pull(Algebra.Output[O](os))
 
   /** Pull that outputs nothing and has result of `r`. */
   def pure[F[x] >: Pure[x], R](r: R): Pull[F, INothing, R] =
-    fromFreeC[F, INothing, R](Result.Pure(r))
+    new Pull(Result.Pure(r))
+
+  val unit: Pull[Pure, INothing, Unit] = new Pull(Result.unit)
```

Comment:
This is redundant with `Pull.done` -- I prefer the name `Pull.unit` but `done` goes back to the old 0.9 API. I'm mildly against adding an alias since I don't like to provide multiple ways of doing the same thing but maybe this is a good exception to that rule considering `done` is kind of a strange name. Thoughts?

Reply:
Oh and if we keep this, can you add ScalaDoc similar to `done`?

Reply:
I think I am going to drop this. I have other changes in mind for `Pull`.